### ISSUE TITLE
Lightbox library (in a plugin) has problems...

### ISSUE THREAD

**Issue by mapkyca** [idno/Known#61]
Actually has a crack at this, but there is *something* in the main template that prevents any of the lightbox/fancybox/bootstrap-lightbox code from functioning. 

Very annoying. 

So perhaps the title of this issue should be "Something about Idno's loaded libraries stuffs up modal dialogs"

**Comment by mck182** [comment]
I've managed to use the MagnificPopup lightbox [1] without a problem. Although I did kind of hardcode it into the template by adding "zoom-gallery" class to the main div of the post. This then nicely creates per post lightbox galleries. I believe it could be done even without that "zoom-gallery" class on the div (maybe even just with the footerJS plugin), but I didn't really bother... :)

[1] - http://dimsemenov.com/plugins/magnific-popup/
